release/models/gnsi/openconfig-gnsi-certz.yang

```diff
@@ -32,6 +32,14 @@ module openconfig-gnsi-certz {
 
   oc-ext:openconfig-version "0.5.0";
 
+  revision 2024-03- {
```

Comment:
```suggestion
  revision 2024-03-05 {
```

Reply:
addressed in https://github.com/openconfig/public/pull/1061/commits/c0ffe5e385876828242f0eb6de7d9f6508d55143